src/features/ModelPlan/ModelToOperations/_components/AddCustomSolutionForm/index.tsx

```diff
@@ -47,7 +47,7 @@ const CustomSolutionForm = () => {
 
   const { modelID } = useParams<{ modelID: string }>();
 
-  const { showMessage, showErrorMessageInModal } = useMessage();
+  const { showMessage, showErrorMessageInModal, clearMessage } = useMessage();
```

Comment:
solution name is not bold in the alert banner after I made a change in a custom solution

![image](https://github.com/user-attachments/assets/761fad58-6b42-455f-af98-770680cc2cfd)